### DISCUSSION TITLE
Call for maintainers

### DISCUSSION THREAD

**Discussion by vinivendra** [vinivendra/Gryphon#125] [General]
Hey folks!

As I've said in [social media](https://twitter.com/vvendra/status/1604583194833354752?s=20), I've recently stopped maintaining Gryphon, and I don't know if and when I'll come back. This means I'm now looking for new maintainers for the project, who can pick up where I left off and continue making sure that Gryphon works well for everybody.

A few people have already contacted me saying they were interested, so I decided to set up this discussion for us to get on the same page and start moving forward. If anyone else wants to join, just let me know here in this thread. The only requirements are that you must be kind to everyone and contribute to Gryphon in some meaningful way during this process. Once we're done, I'll probably transfer this repo into a GitHub organization and give maintainer access to everyone who helped (and wants it), but we can also discuss other ideas.

Before we do that, however, we need to at least get Gryphon up and running again. This means supporting the latest Swift version (currently 5.7.3), fixing any compilation errors that remain, and fixing the test suite so that every test passes (I think some tests will have to be deleted in the process). This thread can also be used to coordinate these efforts. If you're interested in contributing towards these goals, please share with everybody what you're doing so that we avoid duplicated work and can coordinate everything correctly.

Thank you all for your help and understanding so far, and I look forward to working with you 😄

**Comment by laosb** [comment]
Hi, I'm interested in this project from the beginning of my career in Swift based Apple platform development. With about two years of indie experience and a little more than 7 months into full-time job, I'm feeling that I may be somewhat qualified to start contributing in small but meaningful ways.

To be honest, I haven't tried this project more than a simple file yet, and with my last Kotlin project stuck in the pre-COVID era, my experience in Kotlin is getting away from me. Nevertheless, I personally see this opportunity as a challenge for myself to break the barrier into the non-Apple side, and dig deeper both languages.

I may have two to three hours a week to work on this, and I'm feeling I don't really have enough prior knowledge on this (I'm no CS student), but should be enough to do *something* like fixing simple errors and bugs in the upgrade process.

This is an interesting project, and I wish those efforts are not wasted.

**Comment by vinivendra** [comment]
Hi @laosb, thanks for your interest in the project 😄

I've notified other people who were interested in helping, and I've opened issue #127 to track our efforts in getting Gryphon to install and run correctly again. If you'd like to contribute, please join the discussion on that issue, and let me know if you need any help.
